src/components/Dropdown/Dropdown.tsx

```diff
@@ -132,15 +134,25 @@ export const Dropdown: FC<DropdownProps> = ({
         : undefined;
 
     const showClear = !!activeItem && !!onClear;
-
-    const leftPosition =
-        triggerRef.current && overlayRef.current
-            ? triggerRef.current?.getBoundingClientRect().right - overlayRef.current?.getBoundingClientRect().width
-            : "auto";
-    const topPosition = triggerRef.current ? triggerRef.current?.getBoundingClientRect().top - 3 : "auto";
+    const dropdownRef = useRef<HTMLDivElement | null>(null);
+    const placement = `${position}-${alignment}`;
+
+    const popperInstance = usePopper(triggerRef?.current, dropdownRef.current, {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        placement: placement.toLowerCase(),
```

Comment:
By using a map we could bring in the types for placement, what do you think?

```suggestion
    const placementMap: Record<string, VariationPlacement> = {
        ["Top-Start"]: "top-start",
        ["Top-End"]: "top-end",
        ["Bottom-Start"]: "bottom-start",
        ["Bottom-End"]: "bottom-end",
    };

    const popperInstance = usePopper(triggerRef?.current, dropdownRef.current, {
        placement: placementMap[`${position}-${alignment}`],
```